src/web/collectors/ImportsHelperModal.jsx

```diff
@@ -42,11 +35,16 @@ class ImportsHelperModal extends React.Component {
           </Alert>
         </Modal.Body>
         <Modal.Footer>
-          <Button type="button" onClick={this.hide}>Close</Button>
+          <Button type="button" onClick={this.props.onHide}>Close</Button>
         </Modal.Footer>
       </BootstrapModalWrapper>
     );
   }
 }
 
+ImportsHelperModal.propTypes = {
+  showModal: PropTypes.bool.isRequired,
```

Comment:
`PropTypes` import is missing on this component, and that is causing the Collector UI not being loaded.